app/controllers/v0/sign_in_controller.rb

```diff
@@ -225,6 +228,46 @@ def introspect
       render json: { errors: e }, status: :unauthorized
     end
 
+    def fetch_openid_configuration
+      render json: openid_configuration_data
+    end
+
+    def openid_configuration_data
+      openid_data.merge(openid_data_two)
```

Comment:
Why are these split in two structures?

Reply:
Linting would not allow it as whole structure.